src/test/java/DayThreeTest.java

```diff
@@ -24,12 +24,14 @@ public void opCode2MultipliesPostion1And2Together(){
     public void opCode99ImmediatelyHaltsProgramWithMoreThan4Numbers(){
         Assertions.assertThat(dayThree.opcodeProgram("2,4,4,5,99,0")).contains("2, 4, 4, 5, 99, 9801");
     }
+    //what's the difference between this test and the previous one?
     @Test
     public void opCode99ImmediatelyHaltsProgramWithMoreThan4Ns(){
         Assertions.assertThat(dayThree.opcodeProgram("1,0,0,0,99")).contains("2, 0, 0, 0, 99");
         Assertions.assertThat(dayThree.opcodeProgram("1,1,1,4,99,5,6,0,99")).contains("30, 1, 1, 4, 2, 5, 6, 0, 99");
 
     }
+    //not sure this test name is accurate
```

Comment:
Think the name's just been copy-pasted so isn't representative of what's going on.